app/stable_appconfig.go

```diff
@@ -67,8 +67,15 @@ func (app *RegenApp) registerUpgradeHandlers() {
 			"ibc":          1,
 			"genutil":      1,
 			"transfer":     1,
+			"ecocredit":    1, // we dont run InitiGenesis for ecocredit, right?
```

Comment:
so we dont run InitGenesis for ecocredit module, right?

Reply:
```suggestion
			"ecocredit":    0, // Run InitGenesis for ecocredit
```

We should run InitGenesis for ecocredit

Reply:
Correct, because we run it manually, see https://github.com/regen-network/regen-ledger/pull/479#discussion_r691105190

Reply:
```suggestion
			"ecocredit":    1, // we don't run InitGenesis for ecocredit in `RunMigrations`, but manually instead.
```